SConstruct

```diff
@@ -150,6 +203,12 @@ architecture_aliases = {
 }
 opts.Add(EnumVariable("arch", "CPU architecture", "", architecture_array, architecture_aliases))
 
+# Add Verbosity option for compilation
+# opts.Add(BoolVaraible("verbose", "Enable verbose output for the compilation", False))
```

Comment:
The option is commented out, is that intended?

Reply:
It was not I updated the fork, thanks!